hand-written.md

```diff
@@ -41,38 +34,6 @@ Scala 2.12 enables the "GenBCode" backend by default.
 
 The new backend is more efficient than the default backend of Scala 2.11, because it directly generates ASM bytecode from Scala compiler trees, while the previous backend used an intermediate representation called "ICode".
```

Comment:
notes are inconsistent about "backend" vs "back end" vs "back-end". I'd suggest standardizing on either of the latter two, probably "back end".